lib/shopify/resources/customer.ex

```diff
@@ -91,7 +91,12 @@ defmodule Shopify.Customer do
   @doc false
   def send_invite(session, id, %CustomerInvite{} = custom_invite) do
```

Comment:
Since we are already touching this method in this PR, we should rename the variable to customer_invite